aws/docker-pull.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Intended for use on the initial build of a box. Ansible will update/restart
+# if changed subseqsently.
+
+IMAGES=(
+  jrgm/123done
+  mozilla/fxa-auth-db-mysql
```

Comment:
Actually, I'm calling this as good enough. It pulls the default `latest` image, then subsequently, when needed, it will pull and start the container with the configured tag. So, may be slightly slower to create a new `fxa-dev` box, but it will start with what is desired.